docs/onboarding/Configuration-Properties.md

```diff
@@ -78,7 +78,7 @@ The Onboarding Server uses the following public configuration properties:
 | `enrollment-server-onboarding.document-verification.provider` | `mock` | Document verification provider (`mock`, `zenid`). |
 | `enrollment-server-onboarding.document-verification.cleanupEnabled` | `false` | Whether document cleanup is enabled for the provider. |
 | `enrollment-server-onboarding.document-verification.checkInProgressDocumentSubmits` | `0/5 * * * * *` | Cron scheduler for checking status of submitted documents. |
-| `enrollment-server-onboarding.document-verification.verificationOnSubmitEnabled` | `true` | Whether document verification on submit is enabled. |
+| `enrollment-server-onboarding.document-verification.verificationOnSubmitEnabled` | `false` | Whether document verification on submit is enabled. It verifies each document side separately. |
```

Comment:
Can we add a comment that this may break validation in document verification system?